src/pybind/mgr/cephadm/services/osd.py

```diff
@@ -228,8 +228,9 @@ def generate_previews(self, osdspecs: List[DriveGroupSpec], for_host: str) -> Li
                 cmd = self.driveselection_to_ceph_volume(ds,
                                                          osd_id_claims.get(host, []),
                                                          preview=True)
+
                 if not cmd:
-                    logger.debug("No data_devices, skipping DriveGroup: {}".format(
+                    self.mgr.log.info("No data_devices, skipping DriveGroup: {}".format(
```

Comment:
`logger.debug` is correct here, otherwise we're loosing the logger context here.

Reply:
thx! reverted!